roles/wordpress-stack-install.yml

```diff
@@ -1,10 +1,8 @@
 ---
 - name: Install WordPress, MariaDB, Nginx, and PHP-FPM
   hosts: wordpress-server
-  remote_user: root
-  # remote_user: user
-  # sudo: yes
-
+  become: yes
+  become_user: root
```

Comment:
I feel like we are definitely making a bigger deal of it than it needs to be, but it's also good to know proper etiquette. It seems that when you use `hosts:`, you should generally follow it with a `remote_user`. For instance:
```
hosts: wordpress-server
remote_user: <regular user>
become: yes # become root
```
When you wish to change to the `root` user, only `become: yes` is necessary.